usecases/MinimumWorkingExample/dodo.py

```diff
@@ -23,18 +26,27 @@
 #parent directory of the minimum working example
 ParentDir = os.path.dirname(Path(__file__))
 
-# defining paths
+# defining paths for emodule
 emodul_output_directory = Path(ParentDir, 'emodul')  # folder with metadata yaml files
 raw_data_emodulus_directory = Path(ParentDir, 'Data', 'E-modul')  # folder with folders of raw data files
 metadata_emodulus_directory = Path(emodul_output_directory, 'metadata_yaml_files')  # folder with metadata yaml files
 processed_data_emodulus_directory = Path(emodul_output_directory, 'processed_data')  # folder with csv data files
 knowledge_graphs_directory = Path(emodul_output_directory, 'knowledge_graphs')  # folder with KG ttl files
 
+# defining paths for mixture
+raw_data_mixture_directory = Path(ParentDir, 'Data', 'Mischungen')  # folder with raw data files (excel)
+mixture_output_directory = Path(ParentDir, 'mixture')  # folder with folders
+metadata_mixture_directory = Path(mixture_output_directory, 'metadata_yaml_files')  # folder with mixture metadata yaml files
+#processed_data_mixture_directory = Path(mixture_output_directory, 'processed_data')  # folder with csv data files
+mixture_knowledge_graphs_directory = Path(mixture_output_directory, 'knowledge_graphs')  # folder with KG ttl files
+
 # when "cheap option" is run, only this souce of raw data is processed
 cheap_example_name = 'Wolf 8.2 Probe 1'
+cheap_mixexample_name = '2014_12_10 Wolf.xls' # corresponding mixture file
 
 # create folder, if it is not there
 Path(emodul_output_directory).mkdir(parents=True, exist_ok=True)
+Path(mixture_output_directory).mkdir(parents=True, exist_ok=True)
```

Comment:
could you group the stuff related to mix and E (such that all emodul dirs are in one block, and then the mix is another one)

Reply:
at some point we would have to separate that also in different task (with the mix being a precondition for the Youngs modulus), so it would be great to do that already now.